doctr/datasets/sroie.py

```diff
@@ -49,35 +48,36 @@ def __init__(
         self.sample_transforms = sample_transforms
         self.train = train
 
-        if rotated_bbox:
-            raise NotImplementedError
-
-        # # List images
         tmp_root = os.path.join(self.root, 'images')
         self.data: List[Tuple[str, Dict[str, Any]]] = []
         np_dtype = np.float32
+
         for img_path in os.listdir(tmp_root):
+
             # File existence check
             if not os.path.exists(os.path.join(tmp_root, img_path)):
                 raise FileNotFoundError(f"unable to locate {os.path.join(tmp_root, img_path)}")
+
             stem = Path(img_path).stem
-            _targets = []
             with open(os.path.join(self.root, 'annotations', f"{stem}.txt"), encoding='latin') as f:
-                for row in csv.reader(f, delimiter=','):
-                    # Safeguard for blank lines
-                    if len(row) > 0:
-                        # Label may contain commas
-                        label = ",".join(row[8:])
-                        # Reduce 8 coords to 4
-                        p1_x, p1_y, p2_x, p2_y, p3_x, p3_y, p4_x, p4_y = map(int, row[:8])
-                        left, right = min(p1_x, p2_x, p3_x, p4_x), max(p1_x, p2_x, p3_x, p4_x)
-                        top, bot = min(p1_y, p2_y, p3_y, p4_y), max(p1_y, p2_y, p3_y, p4_y)
-                        if len(label) > 0:
-                            _targets.append((label, [left, top, right, bot]))
-
-            text_targets, box_targets = zip(*_targets)
-
-            self.data.append((img_path, dict(boxes=np.asarray(box_targets, dtype=np_dtype), labels=text_targets)))
+                _lines = [[x.strip() for x in line.split(',')] for line in f.readlines()]
```

Comment:
This is the only line that needs to be indented

Reply:
Could you check that your proposition is actually as fast as with `csv` (it's standard library so I'm not sure)

Reply:
I think you're missing a `if len(line) > 0` to reproduce the same behaviour

Reply:
@fg-mindee 
you are right csv is a bit faster :)
i have performed this check by `len(line)>=9 ` but now changed